extensions/amp-social-share/0.1/amp-social-share-config.js

```diff
@@ -13,147 +13,57 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import {user} from '../../../src/log';
 
 /**
  * Get social share configurations by supported type.
  * @param  {!string}
  * @return {!Object}
  */
 export function getSocialConfig(type) {
-  user.assert(type in BUILTINS,
-    'Unknown social share type ' + type);
   return BUILTINS[type];
 }
 
 /**
  * @type {Object<string, Object>}
  */
 const BUILTINS = {
-  'twitter': {
-    'url': 'https://twitter.com/intent/tweet',
-    'text': '\u00A0', // Use a nbsp to ensure the anchor isn't collapsed
-    'params': {
-      'url': {
-        'param': 'url',
-        'required': false,
-        'type': 'url',
-        'maxlength': 1024,
-      },
-      'text': {
-        'param': 'text',
-        'required': false,
-        'type': 'text',
-        'maxlength': 140,
-      },
-      'attribution': {
-        'param': 'via',
-        'required': false,
-        'type': 'text',
-        'maxlength': 20,
-      },
+  twitter: {
+    shareEndpoint: 'https://twitter.com/intent/tweet',
+    defaultParams: {
```

Comment:
I really like defaultParams better because it's easier to read and understand you can override these values. And it's easier to read in the context of the code.